packages/modeling/src/operations/modifiers/edges.js

```diff
@@ -0,0 +1,204 @@
+const vec3 = require('../../maths/vec3')
+const line3 = require('../../maths/line3')
+
+const poly3 = require('../../geometries/poly3')
+
+// an edge is an array of two vertices
+const addEdge = (edges, edge, polygon) => {
+  let ei = edges.findIndex((element) => {
+    if (element) {
+      if (vec3.equals(element[0], edge[0]) && vec3.equals(element[1], edge[1])) return true
+      if (vec3.equals(element[0], edge[1]) && vec3.equals(element[1], edge[0])) return true
+    }
+    return false
+  })
+
+  if (ei < 0) {
+    edge.polygons = [polygon]
```

Comment:
~~If the edge already has polygons, but is just not found in edges, it will wipe out the edges references.  This seems dangerous if we are exposing this function to unknown uses.~~

Nevermind, this one is not exposed.  Worth annotating as a private method somehow, in general, if a file has multiple exported methods?